### ISSUE TITLE
Not detecting device

### ISSUE THREAD

**Issue by vettloffah** [pkviet/obs-asio#9]
The plugin is not detecting my Apollo Twin USB on windows 10 machine.

The "device" list is empty.

![image](https://user-images.githubusercontent.com/3166892/51433148-75a13900-1c12-11e9-8270-3b31c0f5bcf1.png)

Any tips to troubleshoot?

Edit: The plugin worked in the past when I was using a different device, but since I switched to this new one it's not finding it. I have already uninstalled and reinstalled the plugin and the drivers for the device. The device works with my DAW.

**Comment by pkviet** [comment]
i'm adding this device to the unsupported list.
I've tried to help someone with this device. The COM implementation of the driver is faulty somewhere. It can't be loaded by either bassasio library, nor portaudio (which is used in custom builds of Audacity supporting asio). Since those are the libs we use for asio support, there's no solution.
I did extensive troubleshooting with the user who had the apollo twin; he opened a ticket with UAD. But they were not helpful at all.
So I'm adding that device to the list of crap hardware ... 
In order to go further I would need to have an UAD device; but given the price and that I have no use for one, support for the UAD devices won't come anytime. Sorry.
